src/psyclone/tests/core/signature_test.py

```diff
@@ -173,3 +178,64 @@ def test_signature_comparison():
         _ = Signature(("a", "b")) >= (1, 2)
     assert "'>=' not supported between instances of 'Signature' and 'tuple'" \
         in str(err.value)
+
+
+def test_to_fortran():
```

Comment:
Maybe rename to "test_to_language_fortran"?

Reply:
Done.